.gitignore

```diff
@@ -21,6 +21,8 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+.DS_Store
```

Comment:
put a * before .DS_Store since some instances of that file are nested in other folders

Reply:
Plus 1